src/QSymbolicsBase/predefined_fock.jl

```diff
@@ -161,4 +180,42 @@ julia> qsimplify(S*vac, rewriter=qsimplify_fock)
     basis::FockBasis
 end
 SqueezeOp(z::Number) = SqueezeOp(z, inf_fock_basis)
-symbollabel(x::SqueezeOp) = "S($(x.z))"
\ No newline at end of file
+symbollabel(x::SqueezeOp) = "S($(x.z))"
+
+"""Thermal bosonic state in defined Fock basis."""
+@withmetadata struct ThermalState <: AbstractSingleBosonOp
+    photons::Int
```

Comment:
This should probably be a real number. But what if someone wants a symbolic one?